Dfe.PrepareConversions/Dfe.PrepareConversions/Pages/AnnexB/Index.cshtml

```diff
@@ -0,0 +1,50 @@
+@page "/task-list/{id:int}/annex-b-form-saved"
+@using Microsoft.IdentityModel.Abstractions
+@model Dfe.PrepareConversions.Pages.AnnexB.IndexModel
+@{
+   ViewData["Title"] = "Have you saved the Annex B form";
+}
+
+@section BeforeMain
+{
+   <a asp-page="@Links.ProjectList.Index.Page" class="govuk-back-link">@Links.ProjectList.Index.BackText</a>
+}
+
+<partial name="_ProjectHeader" model="@Model.Project"/>
+
+<div class="govuk-grid-row">
+   <div class="govuk-grid-column-full">
+      <dl class="govuk-summary-list">
+
+         <div class="govuk-summary-list__row" data-cy="select-annex-b-form-returned">
+            <dt class="govuk-summary-list__key govuk-!-width-one-third">
+               Have you received a completed Annex B form from the school
```

Comment:
Should there be a ?

Reply:
I agree that there _should_ be, but it isn't present in either the story or the prototype, so I have replicated those precisely.